apps/backend/src/modules/users/user.repository.ts

```diff
@@ -18,10 +28,25 @@ class UserRepository implements Repository {
 				passwordHash,
 				passwordSalt,
 			})
-			.returning("*")
-			.execute();
+			.returning("*");
+
+		const details = await this.userDetailsModel
```

Comment:
```suggestion
		const userDetails = await this.userDetailsModel
```
Let's be more specific